stablehlo/tests/verify_reduce_window.mlir

```diff
@@ -79,24 +79,6 @@ func.func @reduce_window_with_non_scalar_block_arg2(%arg0: tensor<4x2xf32>,
 
 // -----
 
-func.func @reduce_window_invalid_inputs() -> (tensor<2x2xf32>, tensor<2x2xi32>) {
```

Comment:
Why was this test deleted?

Reply:
I mentioned it in another comment but it's buried in the comments :-) Let me put it again here:
As we change the `operands` to `adaptor.getInputs()` and `adaptor.getInitValues()`, The ODS (Trait SameVariadicOperandSize) verifies the inputs and initValues have positive sizes and same length. So that our check is not needed any more. 
This test was trying to catch the case that there're NO arguments at all. This is covered by the ODS checks.
When before, the `verify()` happens before ODS check, so it looked like a necessary test.

Reply:
Got it! Yeah sorry I couldn't find the old comment thread, I thought it had been answered somewhere but had forgotten. Thanks!